app/src/main/java/com/example/mobile/monitors/AudioMonitor.kt

```diff
@@ -48,7 +57,19 @@ class AudioMonitor(): IMonitor {
         // i dB sono calcolati come dBFS dato che stiamo lavorando con segnali digitali
         // reference: https://en.m.wikipedia.org/wiki/DBFS
         val rms = rootMeanSquared(buffer)
-        return decibelFromRms(rms)
+        val decibelValue = decibelFromRms(rms)
+        //TODO filemanager era la vecchia classe perscrivere in file( ora su db) ma il metodo classifyvalue è ancora li-> spostarlo
+        val classification = fileManager.classifyValue(decibelValue,MonitorType.AUDIO)
+        //fileManager.saveData(MonitorType.AUDIO, "$decibelValue ($classification)")
+        val values = ContentValues().apply {
+            put("valore", decibelValue) // inserisci il valore desiderato per il campo 'valore'
+            put("classificazione", classification) // inserisci il valore desiderato per il campo 'classificazione'
+        }

```

Comment:
Spostiamo questa logica in un metodo dentro `DatabaseHelper` così evitiamo di duplicare sto codice tra i monitor e lasciamo tutta la gestione del db in quella classe